administrator/components/com_content/tmpl/articles/default.php

```diff
@@ -74,10 +76,10 @@
     <div class="row">
         <div class="col-md-12">
             <div id="j-main-container" class="j-main-container">
-                <?php
-                // Search tools bar
-                echo LayoutHelper::render('joomla.searchtools.default', ['view' => $this]);
-                ?>
+            <?php
+				// Search tools bar
+				echo LayoutHelper::render('joomla.searchtools.default', array('view' => $this, 'options' => ['selectorFieldName' => 'featured']));
+				?>
```

Comment:
```suggestion
                <?php
                // Search tools bar
                echo LayoutHelper::render('joomla.searchtools.default', array('view' => $this, 'options' => ['selectorFieldName' => 'featured']));
                ?>
```
Spaces and not tabs for indentation.